app/services/vot/authn_context_class_ref_component_values.rb

```diff
@@ -1,35 +1,45 @@
 # frozen_string_literal: true
 
 module Vot
-  module LegacyComponentValues
+  module AuthnContextClassRefComponentValues
     ## Identity proofing ACR values
+
+    # @deprecated - Use IAL1
     LOA1 = ComponentValue.new(
       name: Saml::Idp::Constants::LOA1_AUTHN_CONTEXT_CLASSREF,
-      description: 'Legacy LOA1',
+      description: 'Legacy LOA1 - no identity proofing',
       implied_component_values: [],
       requirements: [],
     ).freeze
+    # @deprecated - Use IAL2
     LOA3 = ComponentValue.new(
       name: Saml::Idp::Constants::LOA3_AUTHN_CONTEXT_CLASSREF,
-      description: 'Legacy LOA3',
+      description: 'Legacy LOA3 - identity proofing is performed',
       implied_component_values: [],
       requirements: [:aal2, :identity_proofing],
     ).freeze
+
     IAL1 = ComponentValue.new(
       name: Saml::Idp::Constants::IAL1_AUTHN_CONTEXT_CLASSREF,
-      description: 'Legacy IAL1',
+      description: 'IAL1 - no identity proofing (rev 3)',
       implied_component_values: [],
       requirements: [],
     ).freeze
     IAL2 = ComponentValue.new(
       name: Saml::Idp::Constants::IAL2_AUTHN_CONTEXT_CLASSREF,
-      description: 'Legacy IAL2',
+      description: 'IAL2 - identity proofing is performed (rev3)',
       implied_component_values: [],
       requirements: [:aal2, :identity_proofing],
     ).freeze
+    IAL2_BIO_REQUIRED = ComponentValue.new(
+      name: Saml::Idp::Constants::IAL2_BIO_REQUIRED_AUTHN_CONTEXT_CLASSREF,
+      description: 'IAL2 - identity proofing with biometric comparison (rev3)',
+      implied_component_values: [],
+      requirements: [:aal2, :identity_proofing, :biometric_comparison, :two_pieces_of_fair_evidence],
+    ).freeze
```

Comment:
Only adding `IAL2_BIO_REQUIRED`, since this [work](https://github.com/18F/identity-idp/pull/10948/files#r1680035939) handles replacing `IAL2_BIO_PREFERRED` with the appropriate IAL ACR value.